web/themes/contrib/civictheme/includes/mobile_navigation.inc

```diff
@@ -55,7 +54,8 @@ function _civictheme_preprocess_block__civictheme_mobile_navigation(array &$vari
       $menu_name = str_replace('menu_block:', '', (string) $block_settings['id']);
       $root = empty(str_replace($menu_name . ':', '', $block_settings['parent'] ?? '')) ? '' : $block_settings['parent'];
 
-      $menu_parameters = new MenuTreeParameters();
+      $menu_tree = \Drupal::menuTree();
+      $menu_parameters = $menu_tree->getCurrentRouteMenuTreeParameters($menu_name);
```

Comment:
getCurrentRouteMenuTreeParameters Checks for the "expanded" flag. 
and loads only those that have been marked as expanded. 

MenuTreeParameters() ignores the flag and loads the full menu tree. 
which results the false flag set within 
buildItems

```
// Set a variable for the <li> tag. Only set 'expanded' to true if the
      // link also has visible children within the current tree.
      $element['is_expanded'] = FALSE;
      $element['is_collapsed'] = FALSE;
      if ($data->hasChildren && !empty($data->subtree)) {
        $element['is_expanded'] = TRUE;
      }
```